StkAutomation/Python/Scenario_Building/README.md

```diff
@@ -49,3 +49,25 @@ This script is a demo of the STK Python API that creates a complex search and re
 * Scenario: N/A
 
 ---
+## [VolumetricSensorCoverage.py](VolumetricSensorCoverage.py)
```

Comment:
I suggest you make a VoumetricSensorCoverage folder and then include the comtypes and pythonapi version there rather than making it file name dependent.

Reply:
If you are going to make these file-name dependent then it should be `VolumetricSensorCoverage_comtypes.py` and `VolumetricSensorCoverage_pythonapi.py` for consistency.